### DISCUSSION TITLE
How to change the website logo?

### DISCUSSION THREAD

**Discussion by iamsantanubanerjee** [thuliteio/doks#677] [Q&A]
By default, it is the file icon. How do I custom set it?

Thanks in advance

**Comment by h-enk** [comment]
Generate favicon files (using e.g.[ favicon.io](https://favicon.io/) — recommended) and copy the files to `./static/`. That's all.

**Reply by pranavdhawale** [reply]
![image](https://github.com/user-attachments/assets/7d1f4198-8a9c-40f3-b8a9-c923a26d3509)
I changed each of the favicon and images in assets folder. It's working on my local machine but its not changing the logo when hosted

![image](https://github.com/user-attachments/assets/d5c383e9-2c48-4689-9e0a-3fa1b5529145)
Here you can see, that on local the logo is changed, but the hosted one's hasn't changed

**Reply by h-enk** [reply]
- In the `static` directory, delete all favicon files (if any).
- In the `assets` directory, delete `favicon.ico` and `mask-icon.svg`, you only need `favicon.png` (and optionally `favicon.svg`). Also delete the latter if not used.

See [Favicons](https://seo.thulite.io/docs/built-ins/favicons/)

**Reply by pranavdhawale** [reply]
I just cleared the cookies and cache of the website and it worked for me. Thanks though!

**Reply by h-enk** [reply]
Super!
